posts/2021-08-30-sharing-data-with-fine-grained-control.md

```diff
@@ -0,0 +1,792 @@
+---
+title: "Sharing Data with Fine-grained Control"
+description: "Defining Data Visibility Constraints Among Partners in a Universal Application"
+longDescription: >
+    Defining Data Visibility Constraints Among Partners in a Universal Application
+date: '2021-08-30'
+authors:
+  - Brian McNamara
+---
+
+Vendia Share offers customers the ability to share file and scalar data with internal and external partners. Up to this point, parties to a Universal Application (Uni) could define controls on _files_ but could not define the same control on world state _scalar_ data. Today, we're proud to announce the release of these controls on world state data. With this release, Uni nodes can specify which party has the ability to read or write data on a per-object basis.
```

Comment:
Question about the intended audience.  Do we believe this will be read mainly by those already familiar with Vendia's capabilities and terminology or by those who may not be?

If that latter, I might rework the first paragraph to use more common terms.  Something like:

Vendia Share offers customers the ability to share data with internal and external partners.  Often, our customers want to share data with partners while also enforcing access controls restricting partners to only the data they're authorized to access.  Today, we're proud to announce the release of this much anticipated feature. With this release, partners can specify which party has the ability to read or write data on a per-element basis.

Reply:
Good call - I've updated it in the latest draft.